requirements.txt

```diff
@@ -4,4 +4,5 @@
 # SPDX-License-Identifier: MIT
 
 Adafruit-Blinka
-adafruit-circuitpython-requests
+adafruit-circuitpython-requests,
+adafruit-requests
```

Comment:
You can also remove this `adafruit-requests` since the package you're trying to import is done just above it.  Also, you'll need to remove the comma on the line above for `pip` to be able to install that package!